@ndlib/gatsby-theme-marble/src/assets/css/global.css

```diff
@@ -1,16 +1,11 @@
 html,
 body {
   margin: 0;
-  overflow: hidden;
-  position: fixed;
+  overflow-x: hidden;
```

Comment:
This works but it's kind of hacky. You could have elements off-screen unintentionally, so I see this as more of a band-aid than a resolution of the true cause. The <main> tag has a `:before`, and I'm not sure what it's purpose is... But in any case, it has:
```
display: 'block',
position: 'absolute',
top: '0',
bottom: '0',
left: '70vw',
width: '100vw',
```
Hence, this is causing it to exceed what should be the right boundary by 70vw.

The header is also calculated with a width of 100vw and 90vw for a child which also causes it to slightly exceed the window size.

Reply:
Changed.  I think on the header witdths are related to the css that webtheme was using to create the gutter.  There are margins on it to make it reach 100.   

I also don't know what that before was for.   The webtheme implementation has several.  Some I have removed.  but since to start I copied the css they were writing and have been simplifying it as I go feel free to ditch any of the remaining ones that are unnecessary.